src/binutil.ts

```diff
@@ -48,31 +49,43 @@ export function execPath(shell : Shell, basePath : string) : string {
 
 type CheckPresentFailureReason = 'inferFailed' | 'configuredFileMissing';
 
-function alertNoBin(host : Host, binName : string, failureReason : CheckPresentFailureReason, message : string) : void {
+function alertNoBin(host : Host, binName : string, failureReason : CheckPresentFailureReason, message : string, installDependencies : () => void) : void {
```

Comment:
I like calling it `installDependencies` here because then I can write `installDependencies();` to invoke it, which I find more readable than `installDependenciesCallback();`.  But I think in the context structure it's better to identify it as a callback.  I take your point about the inconsistency though.